src/main/java/dk/kb/datahandler/kaltura/KalturaUtil.java

```diff
@@ -0,0 +1,110 @@
+package dk.kb.datahandler.kaltura;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.kaltura.client.enums.MediaType;
+
+import dk.kb.datahandler.config.ServiceConfig;
+
+public class KalturaUtil {
+
+
+    private static String DOMS_RADIOTV_PATH;     
+    private static String PRESERVICA_TV_PATH;
+    private static String PRESERVICA_RADIO_PATH;
+    
+  
+    
+   
+    
+    private static final Logger log = LoggerFactory.getLogger(KalturaUtil.class);
+  
+     static {        
+        DOMS_RADIOTV_PATH = ServiceConfig.getConfig().getString("streams.domsRadioTvPath");
+        PRESERVICA_TV_PATH=ServiceConfig.getConfig().getString("streams.preservicaTvPath");
+        PRESERVICA_RADIO_PATH=ServiceConfig.getConfig().getString("streams.preservicaRadioPath");        
+    }
```

Comment:
Again, with new preservica file references these values should be part of the fie_id solr field

Reply:
Same comment as above. First path (mount) is not configured in preservica.